app/models/legal_aid_application.rb

```diff
@@ -650,6 +650,23 @@ def related_proceedings
     proceedings.where(sca_type: "related")
   end
 
+  def biological_parent_relationship?
+    proceedings.any? { |proceeding| proceeding.relationship_to_child.eql?("biological") }
+  end
+
+  def parental_responsibility_order_relationship?
+    proceedings.any? { |proceeding| proceeding.relationship_to_child.eql?("court_order") }
+  end
+
```

Comment:
I think you need to track parental responsibility agreements as well
```ruby
  def parental_responsibility_agreement_relationship?
    proceedings.any? { |proceeding| proceeding.relationship_to_child.eql?("parental_responsibility_agreement") }
  end
```

This will need adding at each stage.

In the mapping there is a requirement for
>Have they stated the supporting evidence for this (follow up questions) is either via court order OR via a PR agreement.

My CCMS code already has something similar.  Happy to pair if needed

Reply:
👍 I have added a new method to the legal_aid_application model parental_responsibility_evidence? and am reporting this on both the digest and the application_details report - do you think this meets the requirement?

Reply:
👍 added new parental_responsibility_court_order_relationship? and parental_responsibility_agreement_relationship? methods to the legal_aid_application model and am reporting both of these on the digest and application details report.